### ISSUE TITLE
Unresolved Reference T

### ISSUE THREAD

**Issue by SamuelCabralCruz** [golang/mock#613]
Just tried out using gomock and faced the issue that generated mock make reference to m.ctrl.T but there is no getter on test reporter t field.

**Comment by SamuelCabralCruz** [comment]
After browsing the code in the master branch, I observed that T is now a public Field.
The problem was that I was using gomock v1.1.1.
I went in my project root folder and did the following: `go mod tidy`
I observed that a require was added to my go.mod file 
`github.com/golang/mock v1.1.1`
I then updated the version manually to 1.6.0 and ran another `go mod tidy` and afterward the autogenerated code was compiling properly.

I don't know if there is a way to specify a minimal version for go mod to be downloaded along with mockgen, but those two dependencies should probably be in sync.
At least, better documentation in the project readme could be useful for neofit users like me.
